zanata-war/src/main/java/org/zanata/dao/TextFlowDAO.java

```diff
@@ -300,4 +302,73 @@ public long countActiveTextFlowsInProjectIteration(Long versionId) {
         return builder.build();
     }
 
+    /**
+     * Return text flows from HProjectIteration#id=sourceVersionId and
+     * HProjectIteration#id=targetVersionId which has matching content
+     * hash and document id.
```

Comment:
Suggest "Return text flows that have matching document id and content between the given source and target version".